react-youtube-adaptive-loading/src/App.js

```diff
@@ -29,40 +29,39 @@ import { DEV_MODE } from './config';
 
 const API_KEY = DEV_MODE ? '' : 'AIzaSyBFTmwce1IplDO3zFslqpJaM5a5KLsIL70';
```

Comment:
It's Google best-practice not to commit active API keys. That said, we should probably align on a solution we can use here for other demos too. Filed https://github.com/GoogleChromeLabs/adaptive-loading/issues/91 for us to think about this after the conference.

Reply:
Yes, I got you.

Actually I got a warning message from GitHub as well. I think we should store those kinds of APIs in the dot env and git ignored.

Reply:
👍